pyramid_swagger/tween.py

```diff
@@ -122,14 +129,16 @@ def swagger_data(_):
 
         if settings.validate_response:
             with validation_context(request, response=response):
-                validate_response(response, validator_map.response)
+                swagger_handler.handle_response(
+                    response,
+                    validator=getattr(op_or_validators_map, 'response', None))
```

Comment:
I think we could remove this `getattr` by adjusting the `validate_response()` to take the `op_or_validators_map` instead, but that seems fine to wait until the response branch.